docs/otel/sampling.md

```diff
@@ -0,0 +1,101 @@
+particular item of telemetry.
+
+## Probability sampling
+
+For more information on how to perform and interpret probability
+sampling based on these attributes, [consult OTEP 235](OTEP235).
+
+### Sampling threshold
+
+The OpenTelemetry sampling decision is defined in terms in terms of a
+threshold and a source of randomness.  The threshold expresses how
+many traces out of `1<<56` (i.e., `math.Pow(2, 56)`) are rejected by
+the sampler, and forms the basis of a rejection test defined by `T <=
+R`.  An item is sampled when the item's sampling threshold is less or
+equal to its randomness.
+
+When determining sampling randomness from a span, implementations
+SHOULD:
+
+- use the `tracestate` OpenTelemetry T-value field (`rh`) if it is present, or
+- use the `sampling.threshold` attribute value if it is present.
+
+Otherwise, no information is available about what sampling was
+performed.
+
+### Sampling randomness
+
+When determining sampling randomness from a span, implementations
+SHOULD:
+
+- use the `tracestate` OpenTelemetry R-value field (`rv`) if it is present, or
+- use the `sampling.randomness` attribute value if it is present, or
+- use the least significant 56 bits of the W3C Trace Context TraceID, as described in the W3C Trace Context Level 2 specification.
+
+Callers SHOULD NOT require trace flags to have the Trace Context Level
+2 Random flag set, in case the Trace ID is used as the source of
+randomness, because it is not reliable information at this time.
+Additionally, the W3C Trace Context Level 2 specification was based on
+the widespread use of at least 56 bits of randomness.
+
+In case the user knowingly creates TraceIDs that do not conform to the
+W3C Trace Context Level 2 specification and they wish to perform
+sampling, they SHOULD synthesize a random R-value and store it in the
+`tracestate` (Spans) or the `sampling.randomness` (Log Records)
+attribute value.
+
+## Span attributes
+
+The following attributes are recognized for Spans.  Since probability
+sampling is tightly coordinated through Context propagation and
+reflected in the W3C Trace Context `tracestate`, which is included in
+the Span data model, it is not common to express sampling randomness
+and threshold using attributes.
+
+<!-- semconv sampling.common(full,tag=otel-span-attributes) -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `sampling.priority` | int | Enables prioritizing and de-prioritizing collection of telemetry. [1] | `1`; `0` | Recommended |
+
+**[1]:** This attribute is useful to express the user's desire to collect telemetry, despite the sampling configuration in place.  Non-zero values MAY prioritize the item for collection independent of probability sampling.  Zero values have the reverse effect and MAY cause telemetry not to be collected.  This convention derives from a conventional tag with the same name in the OpenTracing specification.
```

Comment:
Also, is it helpful to refer to OpenTracing here? Perhaps reference to the OTEP would be more relevant?

Reply:
I've quoted the OpenTracing specification in a normative section and removed it from this location. This attribute has a historical, vague and fixed definition.